index.html

```diff
@@ -3857,6 +3857,72 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section class="informative">
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
```

Comment:
```suggestion
While the base media type for this specification is a JSON-LD document, JSON-LD
```

We only have one base media type.

Reply:
we currently have both `vc+ld+json` and `vp+ld+json`

Reply:
a VP has a structure that is different from VC. Namely, allows the inclusion of `verifiableCredential` ... and allows one to express properties about the VP. Collapsing both things to one concept would ignore that they are different concepts that are to be used for different purposes.